### PR TITLE
🛠 #916 Multiple bug fixes and improvements again

### PR DESCRIPTION
**Pull request by stephanevieira75**
### Add grey-light background when ChannelMenu is opened
https://user-images.githubusercontent.com/36481167/108050847-73f10c00-704a-11eb-8f9d-51e9d0889a8a.mp4

### Redirect member if removed from channel
https://user-images.githubusercontent.com/36481167/108081365-2e940500-7071-11eb-82c3-978a1135049d.mp4

### CODE REVIEW

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/Channel.scss [hunk 1]
```diff
@@ -98,40 +98,44 @@
...
     .user_head .status {
-      border-color: $grey_light;
+      border-color: var(--grey-light);
     }
   }
 
+  &.menu-open {
+    background: var(--grey-light);
+  }
+
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/Channel.scss:112]: Plutôt que faire une nouvelle classe, tu peux faire ça:
`&:hover, &.menu-open`
(et tu met tout le contenu du &:hover)

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/ChannelIntermediate.tsx [hunk 2]
```diff
@@ -64,6 +71,7 @@ export default (props: Props): JSX.Element => {
       visibility={channel.data.visibility || 'public'}
       notifications={notifications.length || 0}
       menu={menu(channel)}
+      menuOpened={menuOpened}
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/ChannelIntermediate.tsx:74]: I definitively prefer "active" instead of "menuOpened"

**Review by RomaricMourgues — CHANGES_REQUESTED** [review]
Perfect, two little changes

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/ChannelIntermediate.tsx (hunk 2 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/ChannelIntermediate.tsx]: Done 👍🏻

File: twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/Channel.scss (hunk 1 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/Channel.scss]: C'est fait, merci :)

**Review by RomaricMourgues — APPROVED** [review]

**Pull request merged**